### DISCUSSION TITLE
Audio-Video Receiver Node

### DISCUSSION THREAD

**Discussion by KSumwalt** [andrei-tatar/node-red-contrib-smartnora#182] [Ideas]
My apologies if I missed seeing it in the available nodes.  I was surprised I did not see one.  I am wondering about the ability to add a node supporting the Audio-Video Receiver device: 
https://developers.google.com/assistant/smarthome/guides/audiovideoreceiver

Kindest Regards,
Karl Sumwalt

**Comment by andrei-tatar** [comment]
@KSumwalt It's there, but it's called `media` node and you need to choose `Audio Video Receiver` as Device Type.

![image](https://user-images.githubusercontent.com/6473183/169091912-fda6cf98-f184-410b-993b-5f73819b5872.png)

**Comment by KSumwalt** [comment]
Sweet!  My apologies for missing that.  Thank you for the reply and all this work!

Karl
